src/schemas/forms/ScriptedCheckSchema.ts

```diff
@@ -0,0 +1,19 @@
+import { z, ZodType } from 'zod';
+
+import { CheckFormValuesScripted, CheckType, ScriptedSettings } from 'types';
+
+import { BaseCheckSchema } from './BaseCheckSchema';
+
+const ScriptedSettingsSchema: ZodType<ScriptedSettings> = z.object({
+  script: z.string().min(1, `Script is required.`),
```

Comment:
Looks like there's an [API validation](https://github.com/grafana/synthetic-monitoring-api/pull/846) for `128kb` on size, do you think we should enforce some sort of limit validation on the frontend?

Reply:
![Screenshot of the Syntehtic Monitoring script editor showing a validation error beneath it as the script is too large.](https://github.com/grafana/synthetic-monitoring-app/assets/6906380/8eaf0efc-7b40-499e-b414-fe21c4d8e5f0)

I have no idea if the way I've implemented this is too naive and we have to account for including imported modules in the limit (which we won't be able to do)? cc @mem